demos/db.default.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace atk4\ui\demo;
+
+// to use MySQL database:
+//   1. copy this file to "db.php"
+//   2. uncomment the line below (and update the configuration if needed)
+//   3. remove the Sqlite code from the new file
+// $db = new \atk4\data\Persistence\SQL('mysql:dbname=atk4_test__ui;host=mysql', 'atk4_test', 'atk4_pass');
+
+$sqliteFile = __DIR__ . '/_demo-data/db.sqlite';
```

Comment:
Can you confirm that the SQL files well remain read-only in "atk-demo" ? We might be running multiple service pods without session affinity.

Reply:
I can confirm that web demo will continue using the DSN env. var, see https://github.com/atk4/ui/pull/1294/files#diff-cdc5998b685321f6451f8c451fcfee93R42

Reply:
Oh ok, so this only affect local demo. Thanks!